### PR TITLE
Sh/beta to prod

### PR DESCRIPTION
**Pull request by shetzel**
### What does this PR do?
All packaging commands from this plugin will become the GA version in the CLI next release.  The commands are aliased so that executing the beta version of them will continue to work.  I.e., executing `force:package:beta:create` will execute the same command (from this plugin) as executing `force:package:create`.

### What issues does this PR fix or reference?
@W-10833244@

### CODE REVIEW

**Review by peternhale — APPROVED** [review]

**Review by WillieRuemmele — APPROVED** [review]
🚀

File: command-snapshot.json [hunk 1]
```diff
@@ -70,16 +70,16 @@
...
-    "alias": []
+    "alias": ["force:package:beta:create"]
   },
   {
-    "command": "force:package:beta:delete",
+    "command": "force:package:delete",
     "plugin": "@salesforce/plugin-packaging",
     "flags": ["apiversion", "json", "loglevel", "noprompt", "package", "targetdevhubusername", "undelete"],
-    "alias": []
+    "alias": ["force:package:beta:delete"]
```
[the remark this answers is not in the document]
Comment (mshanemc) [reply command-snapshot.json:79]: alternatively, once these get sfified we can use the `deprecateAliases` property so they provide warnings for a while

[the remark this answers is not in the document]
Comment (shetzel) [reply command-snapshot.json:79]: Not yet.  I can add one.

[the remark this answers is not in the document]
Comment (shetzel) [reply command-snapshot.json:79]: W-12301372 added

**Comment by WillieRuemmele** [comment]
### QA Notes
---
✅ global search the project for `:beta:` and only found references to the new aliases
✅ verified the aliases were working correctly
✅ `--help` output references prod commands
✅ polling commands reference prod commands
✅ no `beta` directory

📦 🚀

**Pull request merged**